src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+    private final String deadlineBy;
```

Comment:
Consider renaming this constant variable to a name in all uppercase, in adherence to the module's coding standard.